CMakeLists.txt

```diff
@@ -105,6 +105,28 @@ else()
   assert(0 "Unreachable code. Expected optimization level to have been chosen. See misc/Kconfig.")
 endif()
 
+# Dialects of C++, corresponding to the multiple published ISO standards.
+# Which standard it implements can be selected using the -std= command-line option.
+set_ifndef(DIALECT_STD_CPP98 "c++98")
+set_ifndef(DIALECT_STD_CPP11 "c++11")
```

Comment:
Where's c++03?!

(That was sarcasm.  To wit: maybe this is a bit too specific.  You're omitting one very heavily used standard variant, including one that is bleeding edge and still lacks broad compiler support, and even one that isn't expected to be a standard for several years.)

Reply:
C++03 is supported by C++98 flag, please see [C++98 Support in GCC](https://www.gnu.org/software/gcc/projects/cxx-status.html#cxx98)